subprojects/gradle/runner/fake/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id("kotlin")
+    `maven-publish`
+    id("com.jfrog.bintray")
+}
+
+extra["artifact-id"] = "runner-fake"
+
+dependencies {
+    implementation(project(":common:report-viewer"))
+    implementation(project(":gradle:runner:service"))
+    implementation(project(":gradle:runner:shared"))
+}
```

Comment:
Made a new module for fakes for two reasons:
- It shouldn't depend on tests
- It needs to know about the test report. Runner itself shouldn't.